src/commands/init.ts

```diff
@@ -0,0 +1,583 @@
+      char: 'P',
+      description: 'the platform your app needs to run on',
+    }),
+    version: Flags.string({
+      char: 'v',
+      description: 'the version of the platform',
+    }),
+    'build-location': Flags.string({
+      description: "name of the build's location",
+      aliases: ['location'],
+    }),
+    disk: Flags.string({
+      description: 'the name of the disk',
+      char: 'd',
+      dependsOn: ['path'],
+    }),
+    path: Flags.string({
+      description: 'the path where the disk should be mounted',
+      dependsOn: ['disk'],
+    }),
+  };
+
+  public async run(): Promise<void> {
+    const { args, flags } = await this.parse(Init);
+
+    try {
+      await this.setGotConfig(flags);
+
+      this.log(
+        chalk.yellow(`This command interactively creates a basic liara.json configuration file.
+It includes only the essential settings; additional configurations must be added manually.
+📚 For more details on each field and its usage, visit: https://docs.liara.ir/paas/liarajson/.
+
+Afterwards, use liara deploy to deploy your app.
+
+🔑 Press ^C at any time to quit.
+`),
+      );
+
+      this.spinner = ora();
+
+      if (flags.y) {
+        const dirName = path.basename(process.cwd());
+
+        const platform = detectPlatform(process.cwd());
+        const diskConfig = [
+          {
+            disk: 'media',
+            path: '/uploads/media',
+          },
+        ];
+        const configs = this.setLiaraJsonConfigs(
+          getPort(platform) || 3000,
+          dirName,
+          'iran',
+          platform,
+          supportedVersions(platform)?.defaultVersion,
+          diskConfig,
+        );
+        await this.createLiaraJsonFile(configs);
+        this.log(
+          chalk.yellow(
+            "🚫 This file is just a sample file, don't use it for deployment.",
+          ),
+        );
+
+        this.exit(0);
+      }
+
+      const projects = await this.getPlatformsInfo();
+      const appName = await this.promptProjectName(projects, flags.name);
+      const buildLocation = await this.buildLocationPrompt(
+        flags['build-location'],
+      );
+      const platform = await this.findPlatform(
+        projects,
+        appName,
+        flags.platform,
+      );
+      const port = await this.getAppPort(platform, flags.port, projects);
+      const version = await this.promptPlatformVersion(platform, flags.version);
+      const disks = await this.getAppDisks(appName, projects);
+      const diskConfigs = await this.promptDiskConfig(
+        disks,
+        flags.disk,
+        flags.path,
+      );
+      const cron = await this.promptCron(platform);
+      const healthCheck = await this.promptHealthCheck();
+      const configs = this.setLiaraJsonConfigs(
+        port,
+        appName,
+        buildLocation,
+        platform,
+        version,
+        diskConfigs,
+        healthCheck,
+        cron,
+      );
+
+      this.createLiaraJsonFile(configs);
```

Comment:
Don't we need an `await` for this?

```suggestion
      this.createLiaraJsonFile(configs);
```